src/main/java/uk/ac/cam/cl/dtg/isaac/dto/GameboardItem.java

```diff
@@ -57,6 +62,21 @@
      */
     public GameboardItem() {}
 
+    /**
+     * Static factory method for constructing a minimal Gameboard Item from a GameboardContentDescriptor.
+     * Augmentation by mapping the content object to the instance will need to be done separately.
+     * @param contentDescriptor
+     *          a content descriptor to populate the gameboard item's fields.
+     * @return A Minimal gameboard item containing all of the information from the content descriptor.
+     */
+    public static GameboardItem fromContentDescriptorLight(final GameboardContentDescriptor contentDescriptor) {
```

Comment:
The "Light" here is to indicate that it is not a very full GameboardItem, open to better naming suggestions

Reply:
We use "Lightweight" elsewhere for the same idea, so there's probably nothing wrong with this!

Reply:
But `getLightweightItemFromContentDescriptor` or suchlike might be slightly clearer?